diagnostic.md

```diff
@@ -73,13 +88,31 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "POST" "http://www.example.com/donuts/$ID" \
+  --header "Content-Type: application/json" \
+  --data "{
+    \"donut\" : {
+      \"name\": \"French Cruller\",
+      \"price\": \"\$0.99\"
+    }
+  }"
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (donutID) {
```

Comment:
Remember, when you create a new `donut` you don't have an id. the ID gets assigned through the `POST` action.